force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +211,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String strResult = '';
+        Integer counter = 0;
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        for (Integer i = 0; i < repeatCount; i++) {
```

Comment:
(s) solid logic. Try to implement this *without* using the `counter` variable. *Hint - in your loop, the Integer `i` is implicitly keeping track of which iteration you are on.